paper/paper.md

```diff
@@ -0,0 +1,281 @@
+---
+title: 'Connectome Mapper 3: A Flexible and Open-Source Pipeline Software for Multiscale Multimodal Human Connectome Mapping'
+tags:
+  - Python
+  - connectome
+  - multi-scale
+  - MRI
+  - BIDS
+  - BIDS App
+  - containerized workflows
+  - nipype
```

Comment:
I just went through the tags on JOSS to see what tags people use. It's surprising to see `connectome` or `connectomics` return no results yet.
Maybe `neuroscience` can also be a possible tag to add. It yields some relevant results to our paper: https://joss.theoj.org/papers/search?q=neuroscience
`pipeline` or `workflow` tags can also be added.

Reply:
Great suggestion

Reply:
```suggestion
tags:
  - Python
  - neuroscience
  - pipeline
  - workflow
  - connectome
  - multi-scale
  - MRI
  - BIDS
  - BIDS App
  - containerized workflows
  - nipype
```